cmake/app/boilerplate.cmake

```diff
@@ -550,15 +550,19 @@ include(${BOARD_DIR}/board.cmake OPTIONAL)
 # The Qemu supported ethernet driver should define CONFIG_ETH_NIC_MODEL
 # string that tells what nic model Qemu should use.
 if(CONFIG_QEMU_TARGET)
-  if(CONFIG_NET_QEMU_ETHERNET)
-    if(CONFIG_ETH_NIC_MODEL)
+  if(CONFIG_ETH_NIC_MODEL)
+    if(CONFIG_NET_QEMU_ETHERNET)
       list(APPEND QEMU_FLAGS_${ARCH}
-        -nic tap,model=${CONFIG_ETH_NIC_MODEL},script=no,downscript=no,ifname=${CONFIG_ETH_QEMU_IFACE_NAME}
+	-nic tap,model=${CONFIG_ETH_NIC_MODEL},script=no,downscript=no,ifname=${CONFIG_ETH_QEMU_IFACE_NAME}
+      )
+    elseif(CONFIG_NET_QEMU_USER)
+      list(APPEND QEMU_FLAGS_${ARCH}
+	-nic user,model=${CONFIG_ETH_NIC_MODEL},${CONFIG_NET_QEMU_USER_EXTRA_ARGS}
       )
     else()
       message(FATAL_ERROR "
-        No Qemu ethernet driver configured!
-        Enable Qemu supported ethernet driver like e1000 at drivers/ethernet"
+	No Qemu ethernet driver configured!
```

Comment:
This does not work as intended with this change.

Reply:
Yup, Thanks.. Will Fix this..